teacher.rb

```diff
@@ -1,10 +1,8 @@
 require_relative 'person'
 
 class Teacher < Person
-  attr_reader :specialization
-
-  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
-    super(age, name, parent_permission)
+  def initialize(age, specialization, name = 'Unknown')
+    super(age: age, name: name)
     @specialization = specialization
   end
```

Comment:
![image](https://github.com/naanmohammed/oop_school_library/assets/93039359/115f3632-cc2a-43e2-83ce-2d9a06796cdb)

- [ ] It would be nice if you fix this behavior of your app because when I try to create a teacher then it shows the error which you can see in the image above 🆙 Kindly fix this error to make your app work as expected 😊